tests/test_main.py

```diff
@@ -72,11 +77,17 @@ def helper(self, args, policy_file=None, templates=None):
         # FIXME: the load_policy function is not properly restoring current working directory
         self.cwd = os.getcwd()
 
+        if SELINUX_ENABLED:
+            sys.path = [path for path in sys.path if path not in (os.getcwd(), '')]
+
         import selinux
         importlib.reload(selinux)
         import semanage
         importlib.reload(semanage)
 
+        if SELINUX_ENABLED:
+            sys.path = [''] + sys.path
```

Comment:
You ignore `''` at line 81 so I'm not sure this condition is needed...

Maybe you just need to remove `''` from the first ignore condition (line 81)

Thoughts?

Reply:
In the first condition, on line 81, I remove `''` from `sys.path`, because I do not want to import `selinux.py` and `semanage.py` that are present in the test directory (I need to load the system ones).

The reason for adding it back on line 89 is that I don't want to break other imports that may depend on current directory being in `sys.path`. There are no such imports for now, but it seems to me like a good practice. It would probably be clearer, if the code made a backup of `sys.path` and then restored it.

Reply:
IHMO I think made a backup of `sys.path` can be useful and I think adding some few comments to explain your mechanism can be useful in the future for other contributors to help them to better understand the import selinux use case.

Reply:
Thanks for review, I will definitely add a comment so that the mechanism is clear to everyone.

Reply:
You are welcome